docs/software/media.md

```diff
@@ -754,7 +754,7 @@ Also Installs:
 
     E.g. to allow .m4a playback:
 
-    - Since Debian Bullseye (Ampache v5):
+    - Since Debian Bullseye (Ampache v6):
```

Comment:
What about Ampache v5 (below, Ampache v4 is mentioned)?

Reply:
Ampache 5 is never installed, it's either v4 on Buster or v6.